tests/test_external/test_sdk/test_precompilation.py

```diff
@@ -0,0 +1,62 @@
+from netqasm.lang.operand import Template
```

Comment:
This file has great resemblance with examples/sdk_scripts/precompile.py
I guess the purpose is different.

Reply:
You're right. Removed `precompile.py`.